### PR TITLE
Azurite HTTPS Unit Tests

### PR DESCRIPTION
**Pull request by zzhxiaofeng**

### CODE REVIEW

File: tests/blob/authentication.test.ts [hunk 1]
```diff
@@ -22,122 +22,136 @@ configLogger(false);
...
-    }
-  });
-
-  it("Should not work without correct account name @loki @sql", async () => {
-    const serviceURL = new ServiceURL(
-      baseURL,
-      StorageURL.newPipeline(
-        new SharedKeyCredential("invalid", EMULATOR_ACCOUNT_KEY),
-        {
+  [{ prefix: "http" }, { prefix: "https" }].forEach(async testName => {
```
Comment (tzhanl) [inline tests/blob/authentication.test.ts]: prefix -> serverType
testName -> test

**Comment by jongio** [comment]
Can you please run `npm test` and fix the errors?  Thanks.  Also, how are you running these tests locally?

**Comment by zzhxiaofeng** [comment]
Exclude this test temporary since we can't make this pass for https with t1 storage sdk. Will add back while migrate tests to t2 storage sdk. What's more, all these tests can run well locally

**Comment by jongio** [comment]
@zzhxiaofeng - Can you please add a section to https://github.com/jongio/Azurite/blob/master/CONTRIBUTION.md#testing for how to run for https?

**Pull request merged**